app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -61,7 +61,7 @@ object NetworkModule : InjectionModule {
 
         single(named(NOTIFICATION_SERVICE_RETROFIT_QUALIFIER)) {
```

Comment:
I am not sure that we need separate Retrofit instance for NotificationService, cause we don't use base url in requests